### ISSUE TITLE
GroundedSkybox zenit/nadir visual artifacts

### ISSUE THREAD

**Issue by hybridherbst** [mrdoob/three.js#27498]
### Description

The new approach exhibits relatively obvious typical rendering artifacts from "just" sampling a texture on a UVed sphere at the top and bottom:

<img width="597" alt="image" src="https://github.com/mrdoob/three.js/assets/2693840/af3bdb06-799a-4f57-8b1a-8b40d097e74c">
<img width="594" alt="image" src="https://github.com/mrdoob/three.js/assets/2693840/a9b3b4b1-b9ad-4ef2-9c0b-1a88f0ea6d2c">

I think that's why typically a shader is used to sample exact values, instead of relying on UV interpolation to do its thing.

(I understand bringing back a custom shader introduces some of the same complexities on why GroundProjectedSkybox was removed)

cc @elalish 

### Reproduction steps

1. Add GroundedSkybox to a scene with a typical radius
2. Note artifacts

### Code

-

### Live example

-

### Version

r161

**Comment by Mugen87** [comment]
Yeah, these artifacts are inevitable when directly projecting an equirectangular texture onto a sphere geometry. But considering the drawbacks of the previous solution (see https://github.com/mrdoob/three.js/issues/27422#issue-2053326206), I doubt we go back to a based shader approach.

If you are nor happy with these artifacts, it's best if you use the previous version as a custom class in your app.

**Comment by hybridherbst** [comment]
I'm not asking to get the vertex shader back that the previous solution did – the artifacts here would be solved with a fragment shader that ignores the UVs, similar to how .background does that.

**Comment by Mugen87** [comment]
> I'm not asking to get the vertex shader back that the previous solution did – the artifacts here could probably be solved with a fragment shader that ignores the UVs, similar to how .background does that.

Even if you just need a custom fragment shader, we end up with a shader material again. Not using `ShaderMaterial` was a major reason for going towards `GroundedSkybox`, see https://github.com/mrdoob/three.js/issues/27422#issuecomment-1867459020.

**Comment by Mugen87** [comment]
Besides, I personally would use `GroundedSkybox` with special controls settings (e.g. with a maximum polar angle). In this way, the user never sees the artifact at the top. Depending on the loaded 3D object, the bottom artifact isn't visible as well (like in the example) because it is hidden by the object's geometry.

**Comment by elalish** [comment]
If `GroundedSkybox` gets pulled into core it could probably share the .background shader. Not sure that'll happen though.

**Comment by hybridherbst** [comment]
Not sure what the consensus is so far regarding the usage of node-based shaders in examples, but GroundedSkybox could probably use a pretty simple node material to fix the seam issue.
